README.rst

```diff
@@ -47,24 +47,32 @@ Here's an example
 .. code-block:: python
 
     # mycode.py
+    import procrastinate
 
     # Make an app in your code
-    app = procrastinate.App(connector=procrastinate.AiopgConnector())
+    app = procrastinate.App(connector=procrastinate.AiopgConnector(host="localhost", user="postgres", password="password"))
 
     # Open the connection to the database
-    app.open()
+    with app.open():
 
-    # Then define tasks
-    @app.task(queue="sums")
-    def sum(a, b):
-        with open("myfile", "w") as f:
-            f.write(str(a + b))
+        # Automatically apply the `procrastinate` schema to the database
+        try:
+            app.schema_manager.apply_schema()
+        except procrastinate.exceptions.ConnectorException:
+            # Schema has already been applied
+            pass
```

Comment:
I understand the idea to have a single fully functional example, but if we go this way, I'd be much more comfortable to make sure the schema fails early when applied on a database that already has it.

I have a feeling we'll end up making things easier for everyone and implementing a very rudimentary migration engine :D

Reply:
I understand. What would you suggest instead? The following options come to mind:
* not setting up the schema at all
* raising in the `except` block
* setting up the schema with a `try/except` and adding a `NOTE: ` comment above it, that tells the reader that this will fail the second time around and should/must be only done once

Of course I'd be more than happy to get another suggestion from your side.

Reply:
As I said in a [previous comment](https://github.com/peopledoc/procrastinate/pull/368/files#r542528982) I'd like to keep the README snippets simple and to the point. So I would not add the setting up of the schema here in the code.